tests/utils/esx/esxcli.go

```diff
@@ -77,3 +78,39 @@ func WaitForExpectedState(fn getVMPowerStatus, vmName, expectedState string) boo
 	log.Printf("Timed out to poll status\n")
 	return false
 }
+
+// IsVMBackOnline checks if VM is fully back online after being restarted
+func IsVMBackOnline(vmIP, vmName string) bool {
+	isStatusOnline := WaitForExpectedState(GetVMPowerState, vmName, properties.PowerOnState)
+	if !isStatusOnline {
+		log.Printf("VM %s is not in PoweredOn state", vmName)
+		return false
+	}
+
+	isVDVSRunning := isVDVSRunning(vmIP)
+	if !isVDVSRunning {
+		log.Printf("vDVS is not running on VM [%s]. Please check if the previous VM IP [%s] has changed.", vmName, vmIP)
+		return false
+	}
+
+	return true
+}
+
+// isVDVSRunning checks if vDVS is running. This util can be useful
+// in scenarios where vm is powered-on and user wants to find out if
+// vm is fully up to be able to run docker volume commands.
+func isVDVSRunning(ip string) bool {
```

Comment:
No, that shouldn't be a criteria for moving functions around. We have separate files because it's better to keep similar utilities together. 
Also, if someone has to only verify if vDVS is running, then we will have to call IsVMBackOnline() ?
And later on we can add more utils to vDVSUtil.go file